cdap-cli/src/main/java/io/cdap/cdap/cli/commandset/GeneralCommands.java

```diff
@@ -42,6 +43,7 @@ public GeneralCommands(Injector injector) {
         .add(injector.getInstance(ExitCommand.class))
         .add(injector.getInstance(QuitCommand.class))
         .add(injector.getInstance(ConnectCommand.class))
```

Comment:
Is ConnectCommand deprecated now? If so, please mention that in the Class comment for it.